euphonic/cli/intensity_map.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+from typing import List, Optional, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import euphonic
+from euphonic import ureg
+import euphonic.plot
+from euphonic.util import get_qpoint_labels
+from .utils import (_bands_from_force_constants,
+                    get_args, _get_energy_unit, _get_q_distance,
+                    load_data_from_file)
+
+
+_spectrum_choices = ('dos', 'coherent')
+
+
+def get_parser() -> argparse.ArgumentParser:
```

Comment:
I think `intensity-map` should also have an argument to set vmin/vmax, as if there are Bragg peaks it masks everything else, and there may be cases when people want to look at lower intensity modes

Reply:
Done! I don't think we can test the result with the current setup, but if/when we move to image tests then this will be easier.